extensions/notebook/package.json

```diff
@@ -49,6 +49,11 @@
           "items": {
             "type": "string"
           }
+        },
+        "notebook.modifySearchResults": {
```

Comment:
I think something like collapseSearchResults is a better name for this setting (although if someone can come up with something else I'm not completely set on that being the best name either)